R/mcmc-traces.R

```diff
@@ -150,7 +154,8 @@ mcmc_trace <-
            size = NULL,
            np = NULL,
            np_style = trace_style_np(),
-           divergences = NULL) {
+           divergences = NULL,
+           iter1 = 0) {
```

Comment:
I think it makes sense to have the `iter1` argument come right after the `n_warmup` argument.